src/modules/audio/audio.h

```diff
@@ -72,6 +72,7 @@ void lovrAudioGetPose(float position[4], float orientation[4]);
 void lovrAudioSetPose(float position[4], float orientation[4]);
 bool lovrAudioSetGeometry(float* vertices, uint32_t* indices, uint32_t vertexCount, uint32_t indexCount, AudioMaterial material);
 const char* lovrAudioGetSpatializer(void);
+int lovrAudioGetSampleRate();
```

Comment:
Would prefer if sample rate was represented as `uint32_t` everywhere to match miniaudio and `Sound`.